docs/en/engines/table-engines/mergetree-family/annindexes.md

```diff
@@ -66,6 +61,120 @@ returns
    └────┴─────────┘
 ```
 
+## Approximate Vector Search: Quantization With `QBit` {#approximate-nearest-neighbor-search-qbit}
+import ExperimentalBadge from '@theme/badges/ExperimentalBadge';
+<ExperimentalBadge/>
```

Comment:
```suggestion

<ExperimentalBadge/>
```

I think `import ExperimentalBadge from '@theme/badges/ExperimentalBadge';` here needs to be after the frontmatter